<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keeperSystem</class>
 <widget class="QMainWindow" name="keeperSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仓库管理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>仓库信息</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>711</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>商品信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>功能区</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>入库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>删库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>改库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>提交修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
